xonsh/platform.py

```diff
@@ -173,14 +173,14 @@ def pathbasename(p):
 @functools.lru_cache(1)
 def githash():
```

Comment:
While you are here, would you mind c please adding a docstring that mentions this now returns a tuple (more than just the hash), please?